sections/login-form.liquid

```diff
@@ -58,7 +58,9 @@
 
     <hr class="account__separator" />
 
-    <a href="{{ routes.register_url }}" class="account__button account__button--secondary account__divider">{{ section.settings.create_account_label }}</a>
+    {%- if shop.allow_signup -%}
+      <a href="{{ routes.register_url }}" class="account__button account__button--secondary account__divider">{{ section.settings.create_account_label }}</a>
```

Comment:
As I understood this page will also be used for the customers who are not using next-gen shop yet? If so, they won't be able to adjust the text displayed in the link. Does it have default value? Or maybe we should also link it to customer translations?

Reply:
Looking at `kylie-theme/sections/login-form.liquid`, I think it has a default value, right?
```
...
        "label": "Log in button label",
        "default": "Log in"
      }, {
        "id": "create_account_label",
        "type": "text",
        "label": "Create account button label",
        "default": "Create an account"
      }, {
        "id": "reset_password_label",
        "type": "text",
...
```

As for the customer translations, I'm not sure. @zmajstor, @dabrorius, any ideas?

Reply:
In https://github.com/saladdays-nl/booqable/pull/9635 
I've created a controller's concern `ShopNext::UserTranslations` which has methods for user translations, e.g. `user_checkout_translation(key)`

Reply:
And we also have a `user_translations` method in `application_drop.rb`

Reply:
If we want to replace Mustache variables in user translations, we'll need to upgrade `user_translations` method in application_drop.rb (or perhaps include `ShopNext::UserTranslations` module in the application_drop.rb